packages/plugins/typescript/msw/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# @graphql-codegen/typescript-msw
+
+## 1.0.0
+
+### Major Changes
+
+- First release of `typescript-msw` plugin.
+  Generates MSW mock handlers based on GraphQL operations
```

Comment:
We are using changesets for versioning. Please remove the changelog and instead create a changeset as described in https://github.com/dotansimha/graphql-code-generator/pull/6945#issuecomment-955581309